tests/mutation_check.py

```
@@ -0,0 +1,104 @@
+        finally:
+            write(path, original)
+
+        if result is None:
+            print(f"  SKIPPED  {name}: does not compile when broken")
+        elif result.returncode != 0:
+            print(f"  caught   {name}")
+        else:
+            print(f"  MISSED   {name}")
+            missed.append(name)
+
+    subprocess.run(["cmake", "--build", "build", "--target", "volasim_tests", "-j8"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)